apps/spruce/src/pages/version/TestAnalysis/index.tsx

```diff
@@ -0,0 +1,260 @@
+              multiselect
+              onChange={(value: string[]) => {
+                setSelectedTaskStatuses(value);
+                sendEvent({
+                  name: "Filtered test analysis tab",
+                  "filter.by": "task statuses",
+                });
+              }}
+              placeholder="Select a task status"
+              value={selectedTaskStatuses}
+            >
+              {taskStatuses.map((status) => (
+                <ComboboxOption
+                  key={status}
+                  data-cy={`task-status-${status}-option`}
+                  displayName={taskStatusToCopy[status]}
+                  value={status}
+                />
+              ))}
+            </Combobox>
+            <Combobox
+              disabled={!hasResults}
+              label="Build Variant"
+              multiselect
+              onChange={(value: string[]) => {
+                setSelectedBuildVariants(value);
+                sendEvent({
+                  name: "Filtered test analysis tab",
+                  "filter.by": "build variant",
+                });
+              }}
+              overflow="scroll-x"
+              placeholder="Select a build variant"
+              value={selectedBuildVariants}
+            >
+              {buildVariants.map((bv) => (
+                <ComboboxOption
+                  key={bv.buildVariant}
+                  data-cy={`build-variant-${bv.buildVariant}-option`}
+                  displayName={bv.buildVariantDisplayName}
+                  value={bv.buildVariant}
+                />
+              ))}
+            </Combobox>
+          </FilterContainer>
+          {hasResults && (
+            <FilterSubheaderContainer>
+              <Body weight="medium">
+                {totalFilteredTestCount}/{totalTestCount} Failed{" "}
+                {pluralize("Test", totalFilteredTestCount)}
+              </Body>
+              <Button
+                data-cy="clear-filter-button"
+                disabled={!hasFiltersApplied}
+                onClick={() => {
+                  setQueryParams({
+                    [TestAnalysisQueryParams.Statuses]: [],
+                    [TestAnalysisQueryParams.Variants]: [],
+                    [TestAnalysisQueryParams.TestName]: "",
+                  });
+                }}
+                size="xsmall"
+              >
+                Clear Filters
+              </Button>
+              {hasFiltersApplied && <Disclaimer>Filters applied</Disclaimer>}
+            </FilterSubheaderContainer>
+          )}
+
+          <GroupedTestMapList groupedTestsMapEntries={groupedTestsMapEntries} />
+          {!hasMatchingResults && (
+            <BasicEmptyState
+              description=""
+              title={`No ${hasResults ? "matching " : ""}failed tests found`}
+            />
+          )}
+        </>
+      )}
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  margin: 0 ${size.xs};
+`;
```

Comment:
what do you think about increasing the side margin / applying a percentage width on this container? it feels a little bit like things are going far too right into the page